test/helpers/kubectl.go

```diff
@@ -1465,6 +1465,14 @@ func (kub *Kubectl) Delete(filePath string) *CmdRes {
 		fmt.Sprintf("%s delete -f  %s", KubectlCmd, filePath))
 }
 
+// DeleteAndWait deletes the Kubernetes manifest at path filePath and wait
+// for the associated resources to be gone.
+func (kub *Kubectl) DeleteAndWait(filePath string) *CmdRes {
+	kub.Logger().Debugf("waiting for resources in %q to be deleted", filePath)
+	return kub.ExecShort(
```

Comment:
I would use at least `ExecMiddle` here, 10 seconds will probably cause flakes here.

Reply:
Yeah, that makes sense. Updated the command.